### PR TITLE
Removed support for management_page_charset

### LINKED ISSUE #313: Remove support for `management_page_charset`
It does not seem to be a good idea to define different values for `management_page_charset` on different folders in a zope application. This is especially true on Python 3 where [processInputs()](https://github.com/zopefoundation/Zope/blob/master/src/ZPublisher/HTTPRequest.py#L508) converts the inputs to text. The default encoding used is `UTF-8`. There is no way to access the `management_page_charset` when `processInputs()` is called because traversal is done afterwards.

The usage of `management_page_charset` should be replaced by `default-zpublisher-encoding` set in the config file.

Any opinions?

### PR DESCRIPTION
**Pull request by dataflake**
fixes #313

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -20,6 +20,11 @@ Fixes
 - Don't error out when showing permissions for a non-existent user
   (`#437 <https://github.com/zopefoundation/Zope/issues/437>`_)
 
+Other changes
```
Comment (icemac) [inline CHANGES.rst]: I think this is a breaking change as we remove a feature.

Comment (dataflake) [reply CHANGES.rst]: Re-characterized as "Breaking change" in the change log now

Comment (dataflake) [reply CHANGES.rst]: And changed WHATSNEW.rst to reflect the removal.

**Comment by icemac** [comment]
GitHub does not seem to like if the last commit of a PR is skipped from the CI run. It insists that the CI tests have to be run before it can be merged.
I am going to use my administrator rights to merge this PR.

**Pull request merged**